client/src/components/Header/components/NavLinks.js

```diff
@@ -197,6 +164,34 @@ export class NavLinks extends Component {
             <span className='nav-link-dull'>{t('misc.change-theme')}</span>
           )}
         </button>
+        <div className='nav-link nav-link-header' key='lang-header'>
+          {t('footer.language')}
+        </div>
+        {locales.map(lang =>
+          // current lang is a button that closes the menu
+          i18n.language === i18nextCodes[lang] ? (
+            <button
+              className='nav-link nav-link-lang nav-link-flex'
```

Comment:
```suggestion
              className='nav-link nav-link-lang nav-link-flex'
              key={'lang-' + lang}
```

Missing key that I noticed while working on #41004